src/tests/utils/grpc_util_test.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+from unittest import mock
+
+from src.tests.utils.fixtures.mapping_odm_fixtures import build_order_status_history
+from src.utils.grpc_util import (
+    map_mongo_order_status_histories_to_grpc_mongo_order_status_histories,
+)
+
+
+class TestGrpcUtil(unittest.TestCase):
+    @mock.patch("src.utils.grpc_util.get_unix_time_stamp_milliseconds")
+    def test_grpc_util_mapped_mongo_order_status_to_grpc_order_status(
+        self, mocked_get_unix_time_stamp_milliseconds
+    ):
+        mongo_order_status_histories = [
+            build_order_status_history(id="63656f20f2a8a6a247ae31cc"),
+            build_order_status_history(id="63656f20f2a8a6a247ae31cd"),
+        ]
+
+        mapped_to_grpc_mongo_order_status_histories = (
+            map_mongo_order_status_histories_to_grpc_mongo_order_status_histories(
+                mongo_order_status_histories
+            )
+        )
+
+        self.assertEqual(
```

Comment:
Could you assert at least 5 properties to be mapped?

Reply:
Done